TaskListRails/app/assets/stylesheets/application.css

```diff
@@ -20,6 +20,9 @@ a {
   width: 500px;
 }
 
+.simpson_family {
```

Comment:
In CSS we want to separate styles with multiple words with the `-` rather than the `_`